src/component.cy.ts

```diff
@@ -6,9 +6,7 @@ describe('component', () => {
   it('should accept typing', () => {
     const inputValue = 'I will not work';
     cy.mount(html`<my-component></my-component>`)
-
-    cy.get('input').focus().type(inputValue);
-
+    cy.get('input').type(inputValue);
```

Comment:
I think we should keep the `.focus()` here, because otherwise the test would always fail, since the input wouldn't have focus before the `type` call is made, and it will always click.

Reply:
Done. That said, I'd expect 

```js
cy.get('input').type(inputValue);
```

To focus the `input` for me, regardless.